### ISSUE TITLE
Map does not zoom to specified location in simulator

### ISSUE THREAD

**Issue by jamesagada** [codenameone/CodenameOne#2407]
I am trying to put a marker on a map and then show the marker. I am stumped. If I use this code, the marker is first showed and then replaced with a map centred in the US..

`public void showOnMap(Coord location) {
    Form mapDemo = new Form("Maps", new LayeredLayout());
    if (BrowserComponent.isNativeBrowserSupported()) {
        System.out.println("Browser Component Supported");
        MapContainer mc = new MapContainer(MAPS_KEY);
        mapDemo.add(mc);
        Container markers = new Container();
        markers.setLayout(new MapLayout(mc, markers));
        mapDemo.add(markers);

        Coord moscone = new Coord(37.7831, -122.401558);
        //Coord moscone = new Coord(location);
        Button mosconeButton = new Button("");
        mosconeButton.setUIID("Label");
        FontImage.setMaterialIcon(mosconeButton, FontImage.MATERIAL_PLACE);
        markers.add(moscone, mosconeButton);

        mc.zoom(moscone, 5);
        mc.setCameraPosition(moscone);
        //mc.setShowMyLocation(true);
    } else {
        // iOS Screenshot process...
        mapDemo.add(new Label("Loading, please wait...."));
    }
    mapDemo.show();
}
`

**Comment by jamesagada** [comment]
Yes. Just checked. It is resolved. However it seems that when you put a component as a marker, when you move/pan the map, the market stays fixed in its position on the screen rather than moving with the map.

**Comment by shannah** [comment]
I'm not able to reproduce this behaviour.  I'm using the code that you posted in this issue.  Marker moves properly when moving/panning the map.  What platform are you seeing this on?

**Comment by jamesagada** [comment]
I am using NetBeans running on a Mac. If I put in an image rather than a button or other component there is no issue.

**Comment by shannah** [comment]
Can't reproduce.  Make sure you have the latest cn1lib from [here](https://github.com/codenameone/codenameone-google-maps/blob/master/GoogleMaps.cn1lib) (i'm not sure if cn1 settings is updated).  Then refresh cn1libs.  Try a clean build and then run it.  I'm using your exact code, in Netbeans on Mac and it is working fine.

**Comment by jamesagada** [comment]
I refreshed from settings. Ill try from the site provided.

**Comment by jamesagada** [comment]
This issue is still there. I have done a screen cast and sent by email.
